docs/solutions/k8s-on-dcos.md

```diff
@@ -0,0 +1,271 @@
+# Blueprint - Kubernetes on the Mesosphere DCOS
```

Comment:
I made the diagram and it's now out of date. I've filed an issue to track this.

https://github.com/mesosphere/kubernetes-mesos/issues/233